# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xaw3d/redhat.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Xaw3d/XawI18n.h.orig	Wed Jun 20 14:18:34 2001
+++ Xaw3d/XawI18n.h	Wed Jun 20 14:18:50 2001
@@ -27,9 +27,13 @@
 
 ********************************************************/
 
+#if 0
 #ifdef HAS_WCTYPE_H
 #include <wctype.h>
+#if !defined(__linux__)
 #include <widec.h>
+#endif
+#endif
 #define wcslen(c) wslen(c)
 #define wcscpy(d,s) wscpy(d,s)
 #define wcsncpy(d,s,l) wsncpy(d,s,l)
--- Xaw3d/Imakefile.orig	Wed Jun 20 14:14:35 2001
+++ Xaw3d/Imakefile	Wed Jun 20 14:18:50 2001
@@ -10,27 +10,6 @@
 #define IncSubdir X11
 #define IncSubSubdir Xaw3d
 
-#ifdef LinuxArchitecture
-JUMP_ROOT_Xaw3d = $(AWIDGETSRC)3d
-JUMP_IS_HOST_Xaw3d = YES
-JUMP_STUBLIBS_Xaw3d = libXaw3d.sa
-JUMP_SUBDIRS_Xaw3d = $(JUMP_ROOT_Xt)
-JUMP_DIR_Xaw3d = $(JUMP_ROOT_Xaw3d)/shared
-JUMP_VARS_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_vars
-JUMP_FUNCS_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_funcs
-JUMP_IGNORE_Xaw3d = $(JUMP_ROOT_Xaw3d)/jump_ignore
-JUMP_EXPORT_Xaw3d = $(JUMP_DIR_Xaw3d)/jump.vars
-JUMP_IMPORT_Xaw3d = $(JUMP_EXPORT_Xt) $(JUMP_EXPORT_X11)
-JUMP_ADDRESS_Xaw3d = 0x60300000
-JUMP_JUMPTABLESIZE_Xaw3d = 0x4000
-JUMP_GOTSIZE_Xaw3d = 4096
-JUMP_STUBNAMES_Xaw3d = libXaw3d
-JUMP_STUBS_IMPORT_Xaw3d = $(JUMP_STUBS_IMPORT_X11)
-JUMP_LDLIBS_Xaw3d = $(JUMP_ROOT_Xt)/libXt.sa $(JUMP_ROOT_Xt)/libXmu.sa \
-        $(JUMP_ROOT_Xt)/libXext.sa $(JUMP_ROOT_X11)/libX11.sa \
-        $(JUMP_LDLIBS_libc)
-#endif
-
 #ifdef SharedXawReqs
 REQUIREDLIBS = SharedXawReqs
 #endif
@@ -41,7 +20,7 @@
 
 #  EXTRA_DEFINES = -DARROW_SCROLLBAR -DUSEGRAY
 
-#  EXTRA_INCLUDES = -I.
+EXTRA_INCLUDES = -I.
 
 HEADERS = \
 	AllWidgets.h \
